src/b2/DirectDiscImage.cpp

```diff
@@ -6,12 +6,18 @@
 #include "native_ui.h"
 #include "Messages.h"
 #include <limits.h>
+#ifndef B2_LIBRETRO_CORE
 #include "load_save.h"
```

Comment:
I think I can shuffle things around so there's a bit more file I/O stuff in `shared_lib`, then DirectDiscImage can become part of `beeb_lib`, and it should then be easier to reuse outside b2 proper.